src/components/live-collaboration/LiveCollaboration.tsx

```diff
@@ -0,0 +1,38 @@
+import { t } from "../../i18n";
+import { usersIcon } from "../icons";
+import { Button } from "../Button";
+
+import clsx from "clsx";
+import { useExcalidrawAppState } from "../App";
+
+import "./LiveCollaboration.scss";
+
+const LiveCollaboration = ({
```

Comment:
Not sure about the component name, as it will suggest it's an actual live collaboration implementation, or at least does way more.

We should make it clear it's just a button. So `LiveCollaborationDialogButton` or `LiveCollaborationButton`.

Later we may namespace this in a `DefaultButtons` dictionary if we expose more excalidraw buttons.

Reply:
How about `LiveCollaborationtTrigger` ?

Reply:
yeah, let's do it 👍